src/app/components/StakingDateSelector/index.tsx

```diff
@@ -193,15 +167,36 @@ export function StakingDateSelector(props: Props) {
           );
         })}
       </div>
-      <div className="sliderMonth tw-mt-5 tw-pr-0">
-        <Slider {...settingsSliderMonth}>
+      <div className="tw-mt-5 tw-pr-0 tw-relative">
+        <Carousel
+          arrows={false}
```

Comment:
Add key to fix arrow button rendering delay.
```suggestion
          key={selectedYear}
          arrows={false}
```

Reply:
Works like a charm, thanks :)